src/components/offline-activities.tsx

```diff
@@ -1,26 +1,23 @@
 import React from "react";
-import { getActivityDefinition } from "../lara-api";
 import { getOfflineActivities } from "../offline-manifest-api";
-import { Activity, OfflineActivity } from "../types";
+import { OfflineActivity } from "../types";
 import { Header } from "./activity-header/header";
 
 import "./offline-activities.scss";
 
-const OfflineActivityListRow = (props: {activity: OfflineActivity;  onSelectOfflineActivity: (activity: OfflineActivity) => void}) => {
-  const {name, manifestName} = props.activity;
+const OfflineActivityListRow = (props: {activity: OfflineActivity}) => {
+  const {name, manifestName, contentUrl} = props.activity;
   const displayName = manifestName.length > 0 ? `${manifestName}: ${name}` : name;
-  const handleSelectActivity = () => props.onSelectOfflineActivity(props.activity);
   return (
     <tr>
       <td>TDB</td>
-      <td onClick={handleSelectActivity} className="activity">{displayName}</td>
+      <td className="activity"><a href={`?activity=${encodeURIComponent(contentUrl)}`}>{displayName}</a></td>
```

Comment:
I think for this to work right you will need to set `activity=${resourceUrl}&contentUrl=${contentUrl}`